purchase_stock_picking_show_currency_rate/readme/CONFIGURE.rst

```diff
@@ -7,3 +7,14 @@ Rates for non-company currencies must be available and be distinct to 1.0. For e
 #. Go to Invoicing > Configuration > Currencies and go to EUR
 #. Go to Rates smart-button
 #. Update 01/01/2010 record and change rate to 1.5
+
+To set up the date field used for calculating the currency exchange rate on pickings:
+
+#. Navigate to *Inventory > Configuration > Settings*.
+#. Find and set the 'Picking Date Field for Currency Rate' to be used in calculating 
+   the currency exchange rate for completed pickings.
+
+To change the currency rate display style on pickings:
+
+- Go to Accounting (or Invoicing) → Configuration → Settings.
+- In the Picking Rate Display Type section, select the default display type for all currencies.
```

Comment:
It doesn't make much sense to have this config in accounting. It should be moved next to (before) 'Picking Date Field for Currency Rate'.

Please also make the list style consistent with the previous sections.